### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by pedropombeiro**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
This PR adds code to remove watchman watches in the CI server after the build completes, and adds some minor fixes.

Documentation change PR (review please): https://github.com/status-im/status.im/pull/362

status: ready

### CODE REVIEW

File: ci/Jenkinsfile.android [hunk 1]
```diff
@@ -103,6 +103,7 @@ pipeline {
     }
     stage('Cleanup') {
       steps {
+        cmn.nix.shell 'watchman watch-del $(pwd)'
```
Comment (jakubgs) [inline ci/Jenkinsfile.android]: Wouldn't it make more sense to just run this in the `clean` target of our `Makefile`?

**Review by jakubgs — APPROVED** [review]

**Comment by flexsurfer** [comment]
no change for xcode version?

File: ci/Jenkinsfile.android (hunk 1 above)
Comment (pedropombeiro) [reply ci/Jenkinsfile.android]: That's a dev-side scenario and outside of the scope of this PR (though we could add it if we're sure of what we want to do there). This is just to make sure we don't leave new watches running in CI.

**Comment by pedropombeiro** [comment]
> no change for xcode version?

@flexsurfer We could switch to require 10.2 if everyone agrees. I think it makes more sense than requiring 10.1, since it's easier to upgrade than to downgrade. Loosening the requirements to 10.x takes more work, since the default script in nixpkgs doesn't support that.

File: ci/Jenkinsfile.android (hunk 1 above)
Comment (jakubgs) [reply ci/Jenkinsfile.android]: Then at least make it a separate target like `watchman-clean` and call it with `sh 'make watchman-clean'`, othewise you have to add a `script {}` block in the Jenkinsfile.

**Review by jakubgs — APPROVED** [review]

**Comment by pedropombeiro** [comment]
Please review docs in https://github.com/status-im/status.im/pull/362

**Review by cammellos — APPROVED** [review]

**Pull request merged**